brave/browser/brave_browser_context.cc

```diff
@@ -637,6 +637,14 @@ BraveBrowserContext::GetIOTaskRunner() {
       GetPath(), BrowserThread::GetBlockingPool());
 }
 
+base::FilePath BraveBrowserContext::last_selected_directory() {
+      return GetPrefs()->GetFilePath(prefs::kSelectFileLastDirectory);
```

Comment:
we actually have two prefs files. One of them is leftover from electron and is a `PrefRegistrySimple` so it wasn't compatible with other changes, but it still has a few prefs in it and `kSelectFileLastDirectory` is one of them `AtomBrowserContext::RegisterPrefs`. Checking with @bbondy to see if we care about migrating it (in which case we should handle it like `kDownloadDefaultDirectory`), otherwise we should remove it from `AtomBrowserContext`

Reply:
also it should be registered in BraveBrowserContext with `RegisterFilePathPref`

Reply:
sounds like it's fine to just remove the old pref from `AtomBrowserContext`